src/toil/cwl/cwltoil.py

```diff
@@ -1015,7 +1015,9 @@ def main(args=None, stdout=sys.stdout):
     parser.add_argument("--basedir", type=str)
     parser.add_argument("--outdir", type=str, default=os.getcwd())
     parser.add_argument("--version", action='version', version=baseVersion)
-
+    parser.add_argument("--parallel", action="store_true", default=False,
```

Comment:
This already occurs with Toil and will have no affect as the parallel executor isn't on the toil-cwl-runner codepath